.github/workflows/build.yml

```diff
@@ -12,6 +12,14 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
+  QuickFormat:
+    runs-on: ubuntu-latest
+    steps:
+      - name: git-checkout-action
+        uses: actions/checkout@v3
+      - name: dprint-check-action
+        uses: dprint/check@v2.1
+
```

Comment:
New CI Job for better concurrency:

https://github.com/oreboot/oreboot/actions/runs/3371817811/jobs/5594518414#step:3:14